clients/include/testing_proto.hpp

```diff
@@ -0,0 +1,623 @@
+
+// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+// Script-generated file -- do not edit
+
+#pragma once
+
+#include "hipblas_arguments.hpp"
+
+template <typename T>
+inline inline hipblasStatus_t testing_asum(const Arguments& arg);
```

Comment:
are these supposed to be `inline inline`?

Reply:
I think it should be "really inline"  :-) Very good find, more dead code.  That design didn't work as the template prototypes usage didn't work as compiler needed the implementation for the dispatch function pointers, will delete the file.